biz_portal/apps/portal/templates/portal/business_list.html

```diff
@@ -1,8 +1,89 @@
+      <h4 class="mdc-typography--subtitle2 subtitle-clear-search paginator-count mdc-theme--primary">{{ paginator.count }}</h4>
+      <h4 class="mdc-typography--subtitle2 subtitle-clear-search">results</h4>
+    </span>
+
+    {% for business in object_list %}
+    <a href="{{ business.get_absolute_url }}" class="link-to-business-profile">
+      <div class="mdc-card business-results-card">
+        <div class="business-category">
+          <span class="business-category-text">{{ business.status.label }}</span>
+          <span>
+            <i class="material-icons mdc-list-item__graphic mdc-theme--primary" aria-hidden="true">home</i>
+          </span>
+        </div>
+        <div class="business-name mdc-theme--primary">{{ business.registered_name }}</div>
+
+        <hr class="mdc-list-divider">
+
+        <div class="location-and-status-container">
+          <div class="business-details-location">
+              <summary class="business-summary">
+                <span class="business-details-content">
+                  <i class="material-icons mdc-list-item__graphic mdc-theme--primary" aria-hidden="true">location_on</i>
+                  <span class="header-summary">Address here</span>
+                </span>
+              </summary>
+            </div>
+            <div class="business-details-status">
+              <summary class="business-summary">
+                <span class="business-details-content">
+                  <span class="status-icon-span">
+                    <img class="status-icon" src="{% static 'images/ellipse.svg' %}" />
+                  </span>
+                  <span class="type-header">Status:</span>
+                  <span class="header-summary">status label here</span>
+                </span>
+              </summary>
+            </div>
+        </div>
+      </div>
+    </a>
+    {% empty %}
+    <p class="no-businesses-found">No businesses found</p>
+      {% endfor %}
+
+      {% if is_paginated %}
+      <div class="pagination">
+        <span class="page-links">
+          {% if page_obj.has_previous %}
+          <a class="previous-link" href="{% url "business_list" %}?{% param_replace page=page_obj.previous_page_number %}">
+            <button class="mdc-button">
+              <span class="mdc-button__label">Previous</span>
+            </button>
+          </a>
+          {% endif %}
+          <span>
+            Page <span class="page-current mdc-theme--primary">{{ page_obj.number }}</span> of {{ page_obj.paginator.num_pages }}
+          </span>
+          {% if page_obj.has_next %}
+          <a class="next-link" href="{% url "business_list" %}?{% param_replace page=page_obj.next_page_number %}">
+            <button class="mdc-button">
+              <span class="mdc-button__label">Next</span>
+            </button>
+          </a>
+          {% endif %}
+        </span>
+      </div>
+    {% endif %}
+  </div>
+</div>
+
+{% endblock content %}
 
-<h1>Businesses</h1>
 <ul>
```

Comment:
why is there markup outside the content block? Should this be deleted? Is this unused? Don't think it'll render if it's outside the content block.

Reply:
Sorted